src/commands/misc/github/organization.js

```diff
@@ -14,14 +14,13 @@ module.exports = class GitHubOrganization extends Command {
   }
 
   async run ({ t, author, channel, language }, organization) {
-    if (!await this.getOrganization(t, author, channel, language, organization)) throw new CommandError(t('commands:github.subcommands.organization.orgNotFound'))
-  }
-
-  async getOrganization (t, author, channel, language, organization) {
     try {
       channel.startTyping()
       moment.locale(language)
+
       const data = await this.client.apis.github.getOrganization(organization)
+      if (!data.login) throw new Error()
```

Comment:
Remove try-catch block and add error here instead throw new error on catch block.

```suggestion
      if (!data.login) throw new CommandError(t('commands:github.subcommands.organization.orgNotFound'))
```